examples/integration-tutorials/sending-a-simple-message-to-a-service/sending-a-simple-message-to-a-service/README.md

```diff
@@ -0,0 +1,56 @@
+# sending-a-simple-message-to-a-service
+
+This guide demonstrates how to sending a simple message to a ballerina service. This service is basically implemented on real world scenario.Let’s start use case of Healthcare Management System.
+
```

Comment:
I think we need prerequisites to mention the ballerina version. 
This requires the hospital service to be running in the background, so we have to mention that as well.

Reply:
Changed.